### ISSUE TITLE
Optimise Open Source

### ISSUE THREAD

**Issue by f1shy-dev** [benawad/dogehouse#1530]
Ok, so hear me out ok, our operation here on GitHub is quite good, but there is a bit of a problem. This is what our current state of contributing looks like:
![Flowchart](https://cdn.discordapp.com/attachments/810571477770174506/824009676747178004/highquality.png)

Now obviously, what you can see here is a bottleneck mainly by @benawad What's happening is that these react devs want to help, so they start working on `kofta` and `kibbeh`. Now ben says that the new design has to be done, until then no kofta PRs. This causes all the react devs to move to bugs or `kibbeh`, which is the new design. The only problem is that there is nothing to do with the new design, so everyone is confused. 

If we are smart about this, we can plan what needs to be ported over from `kofta` or implemented. Then react devs have stuff to do, while the designer finishes up. Also, the react native app is nearly non-existent, so instead of forcing new-design, we could get people to work on that too.

Overall, we need to take action and do something, so all the react devs here have something to do.

**Comment by samarmohan** [comment]
Yes, I'm getting bored except I'm new so I'm just understanding the code base.

**Comment by benawad** [comment]
0. You are right
1. I want to get more of the new-design done before taking contributions on it so there are design patterns setup that you can use
2. I expect to have a version of the new-design up tomorrow

**Comment by f1shy-dev** [comment]
Hmm ok, are we going to keep `kofta` up until the new design is done, in its current state (maybe some bug fixes)? Because when the new GUI is done `kofta` will be removed/outdated right?

And by *done* do you mean all the components done, the full layout done or? Also, why are we using a fixed position while putting components into place right now?

*(Yeah this is quite a few questions, but I just want to know)*

**Comment by f1shy-dev** [comment]
Can i ask more unrelated questions here or would discord be better?

**Comment by amitojsingh366** [comment]
> 
> 
>     1. You are right
> 
>     2. I want to get more of the new-design done before taking contributions on it so there are design patterns setup that you can use
> 
>     3. I expect to have a version of the new-design up tomorrow

Isn't new design already up? https://next.dogehouse.tv

**Comment by amitojsingh366** [comment]
I know it isn't complete at all but its up

**Comment by samarmohan** [comment]
> Isn't new design already up? https://next.dogehouse.tv

Yes, he wrote it yesterday and today it is up.

**Comment by TheOtterlord** [comment]
The new design is live and replacing kofta!

Kibbeh is now are only concern.
See the [priority list](https://github.com/benawad/dogehouse/issues/1969) for ways to help out.
Triaging issues and fixing bugs is one of the best ways to help out right now.

In addition, since Kofta is gone forever if you find any issues you believe are only relevant to Kofta, please dm me on [discord](https://discord.gg/wCbKBZF9cV)
